framework/platform/glfw_window.h

```diff
@@ -44,6 +44,8 @@ class GlfwWindow : public Window
 
 	float get_dpi_factor() const override;
 
+	float get_retina_factor() const override;
```

Comment:
`Retina` is an Apple concept/brand. This should really just be `get_x_content_scale` as this doesn't necessarily pertain to only macOS, but all platforms.

Reply:
I've changed it to `get_content_scale_factor` because the fact that it's the x factor is part of an implementation detail inside glfw that supports non-uniform scaling (and thus non-square pixels).  I'd rather either support non-uniform scale all the way through (which would require further changes to `Gui` or just assume uniform scaling for now.

Reply:
Okay, I agree with this